.changeset/four-chairs-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Updates `astro add` to add the Tailwind vite plugin instead of the integration
```

Comment:
```suggestion
Updates `astro add` to add the Tailwind Vite plugin instead of the integration
```

Small nitpick I spotted during Talking & Doc'ing